hazelcast-client/src/main/java/com/hazelcast/client/spi/ProxyManager.java

```diff
@@ -100,10 +101,12 @@
  * The ProxyManager handles client proxy instantiation and retrieval at start- and runtime by registering
  * corresponding service manager names and their {@link com.hazelcast.client.spi.ClientProxyFactory}s.
  */
+@SuppressWarnings("checkstyle:classfanoutcomplexity")
 public final class ProxyManager {
-
+    private static final String PROVIDER_ID = "com.hazelcast.client.spi.ClientProxyDescriptorProvider";
```

Comment:
The same style was like like in NodeExtensionFactory:
    private static final String FACTORY_ID = "com.hazelcast.instance.NodeExtension";